feed.xml

```diff
@@ -4,33 +4,32 @@ permalink: /feed.xml
-  >
-<channel>
-    <title xml:lang="en">{{ site.title }}</title>
-    <atom:link type="application/atom+xml" href="http://import.jekyllrb.com/feed.xml" rel="self"/>
-    <link>http://jekyllrb.com{{ site.baseurl }}</link>
-    <pubDate>{{ site.time | date: "%a, %d %b %Y %H:%M:%S %z" }}</pubDate>
-    <lastBuildDate>{{ site.time | date: "%a, %d %b %Y %H:%M:%S %z" }}</lastBuildDate>
-    <language>en-US</language>
-    <description>Import your old &amp; busted site to Jekyll, a blog-aware static site generator.</description>
-    {% for post in site.posts %}
-    <item>
-        <title>{{ post.title | xml_escape}}</title>
-        <link>http://jekyllrb.com{{ site.baseurl | append: post.url }}</link>
-        <pubDate>{{ post.date | date: "%a, %d %b %Y %H:%M:%S %z" }}</pubDate>
-        <dc:creator>{{ post.author }}</dc:creator>
-        {% for tag in post.tags %}<category>{{ tag | xml_escape }}</category>
-        {% endfor %}{% for cat in post.categories %}<category>{{ cat | xml_escape }}</category>
-        {% endfor %}<guid isPermaLink="false">{{ post.id }}</guid>
-        <description><![CDATA[ {{ post.content }} ]]></description>
-    </item>
-    {% endfor %}
-</channel>
-</rss>
+<feed xmlns="http://www.w3.org/2005/Atom"{% if site.lang %} xml:lang="{{ site.lang }}"{% endif %}>
+  <generator uri="http://jekyllrb.com" version="{{ jekyll.version }}">Jekyll</generator>
+  <link href="{{ page.url | prepend: site.baseurl | prepend: site.url }}" rel="self" type="application/atom+xml" />
+  <link href="{{ '/' | prepend: site.baseurl | prepend: site.url }}" rel="alternate" type="text/html" {% if site.lang %}hreflang="{{ site.lang }}" {% endif %}/>
+  <updated>{{ site.time | date_to_xmlschema }}</updated>
+  <id>{{ '' | prepend: site.baseurl | prepend: site.url | xml_escape }}/</id>
+  <title type="html">{{ site.title | smartify | xml_escape }}</title>
+  {% if site.description %}<subtitle>{{ site.description | xml_escape }}</subtitle>{% endif %}
+
+  {% for post in site.plugins %}
+    <entry xml:lang="en_US">
+      <title type="html">{{ post.title | smartify | strip_html | replace: '\n', ' ' | strip | xml_escape }}</title>
+      <link href="{{ post.url | prepend: site.baseurl | prepend: site.url }}" rel="alternate" type="text/html" title="{{ post.title | xml_escape }}" />
+      <id>{{ post.url | prepend: site.baseurl | prepend: site.url }}</id>
+      <content type="html" xml:base="{{ post.url | prepend: site.baseurl | prepend: site.url }}">{{ post.description | strip | xml_escape }}</content>
+      <summary type="html">{{ post.description | strip_html | replace: '\n', ' ' | strip | xml_escape }}</summary>
+      {% comment %}
+      {% assign post_author = post.author | default: post.authors[0] | default: site.author %}
+      {% assign post_author = site.data.authors[post_author] | default: post_author %}
+      {% assign post_author_name = post_author.name | default: post_author %}
+      {% endcomment %}
```

Comment:
This is a Liquid trick. All the `assign` still happens, but it does not output all the newlines and tabs in the output.